gpage.h

```diff
@@ -58,17 +58,17 @@ namespace gcpp {
 
 		//	Copy and move are disabled by const unique_ptr member, but let's be explicit
 		//
-		gpage(gpage&) = delete;
-		void operator=(gpage&) = delete;
+		gpage(const gpage&) = delete;
+		gpage& operator=(const gpage&) = delete;
```

Comment:
FWIW, when I'm =delete-ing copy/move anyway, I just use the shortest spelling that would be recognized as a copy/move and deliberately not-ideal to emphasize that they shouldn't be called.